artiq/coredevice/fastino.py

```diff
@@ -112,7 +113,7 @@ def voltage_to_mu(self, voltage):
         :param voltage: Voltage in SI Volts.
         :return: DAC data word in machine units, 16 bit integer.
         """
-        data = int(round((0x8000/10.)*voltage)) + 0x8000
+        data = int32(round((0x8000/10.)*voltage)) + int32(0x8000)
```

Comment:
A single ``int32(... + 0x8000)`` would be clearer.

Reply:
I agree that would be nicer. Unfortunatly, it does not compile on the coredevice.

Reply:
`int32(int(round(..)) + 0x8000)`

Reply:
Unfortunately, that doesn't compile either.

Reply:
Weird.